changelog/2789.bugfix.rst

```diff
@@ -0,0 +1,2 @@
+Changed the code so we use the typing namespace consistently throughout the
```

Comment:
I don't think we need a changelog entry, this change isn't user-facing?

Reply:
alright, done. :)